src/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj

```diff
@@ -154,10 +154,10 @@
   </ItemGroup>
   <ItemGroup Condition="'$(TargetFramework)' == 'MonoAndroid7'">
     <PackageReference Include="Xamarin.Android.Support.CustomTabs">
-      <Version>25.3.1</Version>
+      <Version>25.4.0.2</Version>
```

Comment:
consider adding a comment stating that the version for CustomTabs and AppCompat needs to be the same

Reply:
done.